### ISSUE TITLE
English translation for the martyrology seems wrong

### ISSUE THREAD

**Issue by jcaudle** [DivinumOfficium/divinum-officium#1086]
I don't think there is anything to do about this other than to look into alternative martyrology translation sources, but I've noticed that the translations we have now are often not in the same order as the Latin or even add or omit passages. Just seemed worth a mention.

**Comment by APMarcello3** [comment]
A part of this issue is that there is not a 1:1 correspondence between the English and the three different Latin versions we have of the Martyrology.

**Comment by jcaudle** [comment]
I figured it wasn’t actually a bug, but it seemed like it was worth a mention. Would it be worthwhile to have me do some work looking through the Latin texts and trying to reconcile the English with them?

**Comment by fiapps** [comment]
If I'm not mistaken, the English translation comes from the mid nineteenth century Marquis of Bute translation, which corresponds to the martyrology as it was then read in England. As changes were made to the calendar, saints were reordered or moved to other days, and new saints were added. There are two ways to approach the problem.

1. Without changing any code, I think it would be possible to add additional English versions corresponding to the other two versions of the martyrology. These could be constructed by copying and pasting entries into the correct order. The new saints would either be left out or translated afresh.

2. On the principle DRY (don't repeat yourself), a better approach to the martyrology would be to use something like the existing inclusion mechanism to reorder the *elogia* (praises of the saints), or even to make the various versions simply a list of *elogia* to be included from data files. This would require new code, including a way to refer to the *elogia.* The latter approach  would also have the advantage of making it easy to supply the various versions in each language.

Either approach requires a lot of time, since it means editing a book-sized corpus of text.

**Comment by MatthaeusRuber** [comment]
In the martyrologium in Latin we can read:

Solémnitas sancti Ioseph Opíficis, Sponsi beátæ Maríæ Vírginis, Confessóris, opíficum Patróni.


But in Italian:

Il natale dei beati Filippo e Giàcomo, Apostoli. Di essi Filippo, dopo aver convertito quasi tutta la Sazia alla fede di Cristo, da ultimo, presso Geràpoli, città dell'Asia, confitto in croce ed oppresso con sassi, si riposò con una fine gloriosa; Giàcomo poi, il quale è detto anche fratello del Signore e primo Vescovo di Gerusalémme, precipitato dalla sommità del tempio, gli si ruppero le gambe, e percosso in testa con un palo da lavandaio, morì, e fu ivi sepolto, non lontano dal tempio.


The same in English:

The blessed Apostles Philip and James. Philip brought nearly all Scythia to believe in Christ, but in the end, at the city of Hierapolis, in Asia, was fastened to a cross and stoned, and so fell asleep gloriously, [in the year 61.] James, who is written of as the brother of the Lord, and was the first Bishop of Jerusalem, was cast down from a pinnacle of the temple, whereby his legs were broken, and he was slain by a blow upon the head from a fuller's pole, and buried there not far from the temple.


The version is Rubrics 1960 - 1960


For my understanding, that version uses the Calendar of 1960 that I can find into the Antiphonary of 1960.


In Antiphonarium Romanum 1960 I can read at page XXII in Kalendarium Perpetuum at May first:

SS. Philippi et Jacobi apostolorum duplex II classis.


However, at the end of April, I can read:

Feria IV infra II post octavam Paschae solemnitas s. Joseph sponsi bmv etc...


It means that in 1960, the saint Joseph feast was during the third weeks of Easter (we are at the fifth) and on May first the Church celebrated saint Philip and James.


For my understanding, who prays the office according with the extraordinary Roman liturgy, except in USA, has to follow the 1960's martyrologium.

I don't know when the Saint Joseph feast was established.

Can you verify, please?

I hope, my letter can be helpful to improve this wonderful project.

**Comment by fiapps** [comment]
> For my understanding, that version uses the Calendar of 1960 that I can find into the Antiphonary of 1960.

Books of chant required a lot of work to update, so outdated material continued to be printed. It was assumed that people using the books would know about recent changes to the rubrics and calendar and adjust accordingly. Breviaries and Missals are a more reliable guide to the calendar.

In 1955, Pius XII added the feast of St. Joseph the Worker to the general calendar on May 1. This is in addition to the feast on March 19, which is often transferred to avoid Holy Week and the Octave of Easter. The latter is the feast you found celebrated in April.

The reason why the English and Italian translations of the martyrology do not include the feast of St. Joseph the Worker is that they were produced before 1955. Steps could be taken to fix this, but it looks like a lot of work, and so far no one has attempted it.

**Comment by MatthaeusRuber** [comment]
Thank you! I am still not an expert and it is surely true that Pope Pius XII in 1955 decided the st. Joseph feast, but this Antiphonarium was published in 1960! 
This is the book:
https://archive.org/details/liber-antiphonarius-1960/page/n33/mode/1up
I don't understand!

**Comment by fiapps** [comment]
Because it was expensive in the last century to manually set type and make engravings of chant, many pages were simply reprinted as-is. So while this book may have been printed in 1960, the [frontispiece](https://archive.org/details/liber-antiphonarius-1960/page/n2/mode/1up) still has the date 1949 and the following page says "Copyright 1924".

**Comment by fiapps** [comment]
I don't think you will find a more recent one. The Church Music Association of America has the same file or a similar one on their [resources page](https://www.churchmusicassociation.org/resources/resource-list-complete/). I believe that between 1961 and 1965 people were still using older texts and adjusting accordingly. With Vatican II planned or in progress it was likely that some further change would be made, so no publisher would undertake the expense of preparing a completely new edition. People bought these books that were reprinting pages from previous editions and made the necessary adjustments. The _editiones typicae_ were of the Breviary and Missal, and other publishers also typeset new editions of these, but not of the chant books. So you need to use a Breviary to see what the Divine Office should be, and then find the corresponding texts in the Antiphonale.

**Comment by MatthaeusRuber** [comment]
Yes! I understood it.

**Comment by FAJ-Munich** [comment]
For a thorough explanation of many of the changes that happened to the Breviary, Missal, and Calendar I can only recommend the "Compendium of the Breviary Reforms" series by Gregory de Pippio published on NewLiturgicalMovement. 

With regards to S. Joseph, there has always been the feast on March 19 and, at least since the 19th century, the Eastertide solemnity of S. Joseph, Universal Patron of the Church! It's the latter feast which has been replaced by Pius XII by the May 1st one displacing Ss. Phillip and James App from their traditional place. 

In 1956, the Apostels were displaced to the first free day, i.e., May 11. That all those pre-council reforms have been carried out in haste and with ignorance to the rubrics and their logic can be seen that the Missal of 1960 keeps them on May 11 despite the abolishment of the May 3rd feast of the Invention of the Cross. So by the very rules of the calendar itself, in 1960, the Apostels should have been brought forth to May 3rd but they weren't. 

What shall I say?

**Comment by MatthaeusRuber** [comment]
Thank you for your explanation. I love to learn new things. Thank you, again!
